script.js

```diff
@@ -148,6 +154,7 @@ let isTimerSet = false;
 let milliSecCountDown = 0;
 let timerIntervalCountDownTimerId = null;
 let hasPausedCountDownTimer = false;
+let countDownTimerStarted = false;
```

Comment:
As mentioned before, if you use the existing variable i.e., `hasPausedCountDownTimer` you will not be needing an additional variable...

Try using the existing variable before creating a new variable.